integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -496,6 +496,15 @@ case class LoadTable(
         val dictFolderPath = carbonTablePath.getMetadataDirectoryPath
         val dimensions = carbonTable.getDimensionByTableName(
           carbonTable.getFactTableName).asScala.toArray
+        if (null == carbonLoadModel.getLoadMetadataDetails) {
+          CommonUtil.readLoadMetadataDetails(carbonLoadModel, storePath)
```

Comment:
Can use single pass for first load, but it will automatically fallback to twopass, which should be logged and continue.